src/components/Issuance/EmailIssuance.tsx

```diff
@@ -24,57 +24,86 @@ import { checkEcosystem  } from '../../config/ecosystem';
 	const inputRef = useRef(null);
 	const [mounted, setMounted] = useState<boolean>(false)
+	const [schemaType, setSchemaType]= useState<SchemaTypes>();
+	const [credentialType, setCredentialType] = useState<CredentialType>();
+	const [credDefId, setCredDefId] = useState<string>();
+	const [schemasIdentifier, setSchemasIdentifier] = useState<string>();
+	const [schemaTypeValue, setSchemaTypeValue] = useState<SchemaTypeValue>();
 
 	const getSchemaCredentials = async () => {
 		try {
 			setLoading(true);
 			const orgId = await getFromLocalStorage(storageKeys.ORG_ID);
-			if (orgId) {
-				const response = await getSchemaCredDef(SchemaType.INDY);
+			const orgDid = await getFromLocalStorage(storageKeys.ORG_DID);
+			let currentSchemaType = schemaType;
+	
+			if (orgDid?.includes(DidMethod.POLYGON)) {
+				currentSchemaType = SchemaTypes.schema_W3C;
+				setSchemaTypeValue(SchemaTypeValue.POLYGON)
+				setCredentialType(CredentialType.JSONLD)
+				
+			} else if ( orgDid?.includes(DidMethod.KEY) || orgDid?.includes(DidMethod.WEB)) {
+				currentSchemaType = SchemaTypes.schema_W3C;
+				setSchemaTypeValue(SchemaTypeValue.NO_LEDGER)
+
+				setCredentialType(CredentialType.JSONLD)
+			}
+				else if (orgDid?.includes(DidMethod.INDY)) {
+				setCredentialType(CredentialType.INDY)
+				currentSchemaType = SchemaTypes.schema_INDY;
+			}
+
+			setSchemaType(currentSchemaType); 
+			if (currentSchemaType && orgId) {
+				const response = await getSchemaCredDef(currentSchemaType); 
 				const { data } = response as AxiosResponse;
 
-				if (data?.statusCode === apiStatusCodes.API_STATUS_SUCCESS) {
-					const credentialDefs = data.data;
+		if (data?.statusCode === apiStatusCodes.API_STATUS_SUCCESS) {
+			const credentialDefs = data.data;
 
-					const options = credentialDefs.map((credDef: ICredentials) => ({
-						value: credDef.credentialDefinitionId,
-						label: `${credDef.schemaName} [${credDef.schemaVersion}] - (${credDef.credentialDefinition})`,
-						schemaName: credDef.schemaName,
-						schemaVersion: credDef.schemaVersion,
-						credentialDefinition: credDef.credentialDefinition,
-						schemaAttributes:
-							credDef.schemaAttributes &&
-							typeof credDef.schemaAttributes === 'string' &&
-							JSON.parse(credDef.schemaAttributes),
-					}));
-					setCredentialOptions(options);
-				} else {
-					setSuccess(null);
-					setFailure(null);
-				}
-				setLoading(false);
-			}
+	const options = credentialDefs.map((credDef: ICredentials) => ({
+		value: schemaType===SchemaTypes.schema_INDY ? credDef.credentialDefinitionId : credDef.schemaVersion,
+		label: `${credDef.schemaName} [${credDef.schemaVersion}]${currentSchemaType === SchemaTypes.schema_INDY ? ` - (${credDef.credentialDefinition})` : ''}`,
+		schemaName: credDef.schemaName,
+		schemaVersion: credDef.schemaVersion,
+		credentialDefinition: credDef.credentialDefinition,
+		schemaIdentifier:credDef.schemaIdentifier,
+		credentialDefinitionId:credDef.credentialDefinitionId,
```

Comment:
We can use object destructuring here to fetch the attributes of the credDef object